### PR TITLE
Release/0.2.0

### PR DESCRIPTION
**Pull request by matus-tomlein**
## Changelog

**Enhancements:**

* Update session when events are tracked (#35)
* Allow custom User Agent strings (#23)
* Use snowplow namespace (#13)
* Add support for C++17 (#36)
* Update sqlite3 to latest version due to use of deprecated APIs (#24)
* Add API documentation (#40)

**Under the hood:**

* Update copyright notices (#26)
* Fix links to documentation in README (#25)
* Build and release using Github actions (#27)
* Add performance testing setup (#37)
* Follow a code style guide (#44)
* Change all OS-X references to macOS (#11)

## Documentation

PR with changes in documentation is in the [data-value-resources repository](https://github.com/snowplow-incubator/data-value-resources/pull/50).

### CODE REVIEW

**Comment by snowplowcla** [comment]
Thanks for your pull request. Is this your first contribution to a Snowplow open source project? Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

:memo: Please visit https://docs.snowplowanalytics.com/docs/contributing/contributor-license-agreement/ to learn more and sign.

Once you've signed, please reply here (e.g. I signed it!) and we'll verify. Thanks.

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,18 @@
+Version 0.2.0 (2022-02-23)
```
Comment (AlexBenny) [inline CHANGELOG]: Sorry, I delayed your release date! :(

**Review by AlexBenny — APPROVED** [review]
👍 🚢 
Sorry, I delayed your release and the Changelog date is old now!

**Comment by matus-tomlein** [comment]
No worries and thanks a lot for all the reviews @AlexBenny!

**Pull request merged**